extensions/http/src/test/java/org/mule/extension/http/BasicHttpTestCase.java

```diff
@@ -44,6 +44,8 @@
 import org.junit.Test;
 
 //TODO: MULE-9702 Remove once the tests are migrated.
+@ArtifactClassLoaderRunnerConfig(plugins = {"org.mule.modules:mule-module-sockets"},
+    providedInclusions = "org.mule.modules:mule-module-sockets")
```

Comment:
Why is needed to define the same artifact both on plugins and providedInclusions fields?